runtime/autoload/man.vim

```diff
@@ -310,20 +308,33 @@ function! s:format_candidate(path, psect) abort
 endfunction
 
 function! man#init_pager() abort
-  " Remove all backspaced characters.
-  execute 'silent keeppatterns keepjumps %substitute,.\b,,e'.(&gdefault?'':'g')
   if getline(1) =~# '^\s*$'
     silent keepjumps 1delete _
   else
     keepjumps 1
   endif
   " This is not perfect. See `man glDrawArraysInstanced`. Since the title is
   " all caps it is impossible to tell what the original capitilization was.
-  let ref = substitute(matchstr(getline(1), '^[^)]\+)'), ' ', '_', 'g')
+  " But it's usually lowercase, so we'll stick to that.
+  let ref = tolower(substitute(matchstr(getline(1), '^[^)]\+)'), ' ', '_', 'g'))
   try
     let b:man_sect = man#extract_sect_and_name_ref(ref)[0]
   catch
     let b:man_sect = ''
   endtry
   execute 'silent file man://'.fnameescape(ref)
 endfunction
+
+function! man#init_highlight_groups() abort
+  let group = 'Keyword'
+  while 1
+    let hl =  execute('highlight '.group)
+    let values = matchstr(hl, '\%(.*xxx\)\?\%(.*cleared\)\?\s*\zs.*')
+    if values !~# '^links to'
+      break
+    endif
+    let group = matchstr(values, '\w\+$')
+  endwhile
+  execute 'highlight default manBold' values 'cterm=bold gui=bold'
```

Comment:
> It looks like spaces are getting remove somehow.

Yes, that's a regression between Vim and Neovim's implementation of execute.

Vim:

```
:echo execute('hi Normal')
Normal         xxx ctermfg=255 ctermbg=235 guifg=#eeeeee guibg=#262626
```

Neovim:

```
:echo execute('hi Normal')
Normalxxx ctermfg=255ctermbg=235guifg=#eeeeeeguibg=#262626
```

It's related to how the highlighting is handled.  The highlighting obviously can't be represented in the resulting string, but for some reason we're also stripping whitespace in that handling.